tests/migration/migration.go

```diff
@@ -2725,7 +2725,19 @@ var _ = SIGMigrationDescribe("VM Live Migration", decorators.RequiresTwoSchedula
 
 			By("Creating ResourceQuota with enough memory for the vmi but not enough for migration")
 			resourceQuota := newResourceQuota(resourcesToLimit, testsuite.GetTestNamespace(vmi))
-			_ = createResourceQuota(resourceQuota)
+			resourceQuota = createResourceQuota(resourceQuota)
+			Eventually(func(g gomegatypes.Gomega) error {
```

Comment:
```golang
			Eventually(func(g gomegatypes.Gomega) {
				quota, err := virtClient.CoreV1().ResourceQuotas(resourceQuota.Namespace).Get(context.TODO(), resourceQuota.Name, metav1.GetOptions{})
				g.Expect(err).ToNot(HaveOccurred())
				for key, value := range resourcesToLimit {
					g.Expect(quota.Status.Hard).To(HaveKeyWithValue(key, value))
				}
			}).WithTimeout(time.Minute).WithPolling(time.Second).Should(Succeed())
```
?

Reply:
I am not sure, we probably need custom compare just in case.

Reply:
From my understanding gomega uses `Equal` func for equality comparison.
The `Equal` func for quantity is defined exactly as:
```
func (q Quantity) Equal(v Quantity) bool {
	return q.Cmp(v) == 0
}
```
I don't think it's something blocking anyway :slightly_smiling_face:

Reply:
It is using reflect.DeepEqual

Reply:
Great to know! Thank you!